Tasks/Xcode/Tests/L0.ts

```diff
@@ -575,4 +575,22 @@ describe('Xcode L0 Suite', function () {
 
         done();
     });
+
+    it('Tests with failures', (done: MochaDone) => {
```

Comment:
I think this could be more specific: "Failed test task should publish results" or something